lib/command/run.rb

```diff
@@ -451,7 +452,7 @@ def resolve_job_status # rubocop:disable Metrics/MethodLength
         Shell.debug("JOB STATUS", status)
 
         case status
-        when "active"
+        when "active", "pending"
```

Comment:
Some jobs may have the status as `pending` before moving to `active`, so we need to wait for those as well.

Reply:
👍